content/cli/v6/using-npm/registry.md

```diff
@@ -15,7 +15,7 @@ package info.
 
 npm is configured to use npm, Inc.'s public registry at
 <https://registry.npmjs.org> by default. Use of the npm public registry is
-subject to terms of use available at <https://www.npmjs.com/policies/terms>.
+subject to terms of use available at <https://docs.npmjs.com/policies/terms>.
```

Comment:
This needs to be updated in https://github.com/npm/cli/blob/v6/docs/content/using-npm/registry.md